presto-docs/src/main/sphinx/presto-cpp.rst

```diff
@@ -0,0 +1,55 @@
+**********
+Presto C++
+**********
+
+Note: Presto C++ is in active development. See :doc:`Limitations </presto_cpp/limitations>`.
+
+.. toctree::
+    :maxdepth: 1
+
+    presto_cpp/features
+    presto_cpp/limitations
+    presto_cpp/properties
+
+Overview
+========
+
+Presto C++, sometimes referred to by the development name Prestissimo, is a C++ 
+drop-in replacement for Presto workers based on the `Velox <https://velox-lib.io/>`_ library. It 
+implements the same RESTful endpoints as Java workers using the Proxygen C++ 
+HTTP framework. 
+Because communication with the Java coordinator and across workers is only 
+done using the REST endpoints, Presto C++ does not use JNI and is commonly 
+deployed in such a way as to avoid having a JVM process on worker nodes.
+
+C++ based Presto's codebase is located at `presto-native-execution
+<https://github.com/prestodb/presto/tree/master/presto-native-execution>`_.
+
+Motivation and Vision
+=====================
+
+Presto aims to be the top performing system for data lakes. To achieve this 
+goal, the Presto community is working toward moving the Presto 
+evaluation engine from the native Java-based implementation to a new 
+implementation written in C++ using `Velox <https://velox-lib.io/>`_.  
+
+By moving the evaluation engine to a library, the intent is to enable the 
+Presto community to focus on more features and better integration with table 
+formats and other data warehousing systems.
+
+Over time, we intend to completely deprecate the Java-based evaluation engine. 
+This cut-off will be signified by a major release update in Presto's 
+versioning scheme to 2.0. After 2.0, new developments will be exclusively 
```

Comment:
This is strange given that Presto is not yet 1.0

Reply:
This is my mistake--the details on versioning, and any cutoff, should first go into an RFC so the community coalesces around the approach.  @steveburnett let's remove this paragraph, thanks!